gems/pending/spec/postgres_ha_admin/failover_databases_spec.rb

```diff
@@ -1,132 +1,93 @@
+    after do
+      @connection.exec("ROLLBACK")
+      @connection.finish
     end
 
-    it "create yaml file with list of available databases if file did not exist" do
-      expect(File.exist?(subject.yml_file)).to be false
+    it "updates 'failover_databases.yml'" do
+      @failover_databases.update_failover_yml(@connection)
 
-      subject.all_databases(connection)
-      expect(File.exist?(subject.yml_file)).to be true
-    end
-
-    it "if yaml file exists, it loads list of databases from existing file" do
-      list = subject.all_databases(connection)
-      expect(list.size).to be 3
+      expect(@failover_databases.active_databases).to contain_exactly(
+        {:type => 'master', :active => true, :host => '1.1.1.1', :user => 'root', :dbname => 'vmdb_test'},
+        {:type => 'standby', :active => true, :host => '2.2.2.2', :user => 'root', :dbname => 'vmdb_test'})
 
       add_new_record
 
-      subject.all_databases(connection)
-      expect(list.size).to be 3
-      expect(list).to include({:type => "master", :host => "1.1.1.1", :dbname => "vmdb_test",
-                              :user => "root", :active => true},
-                              {:type => "standby", :host => "2.2.2.2", :dbname => "vmdb_test",
-                               :user => "root", :active => true},
-                              {:type => "standby", :host => "3.3.3.3", :dbname => "vmdb_test",
-                               :user => "root", :active => false})
-    end
-  end
-
-  describe "#refresh_databases_list" do
-    it "override existing yaml file and returns updated list of databases" do
-      list = subject.all_databases(connection)
-      expect(list.size).to be 3
+      @failover_databases.update_failover_yml(@connection)
+      expect(@failover_databases.active_databases).to contain_exactly(
+        {:type => 'master', :active => true, :host => '1.1.1.1', :user => 'root', :dbname => 'vmdb_test'},
+        {:type => 'standby', :active => true, :host => '2.2.2.2', :user => 'root', :dbname => 'vmdb_test'},
+        {:type => 'standby', :active => true, :host => '4.4.4.4', :user => 'root', :dbname => 'some_db'})
 
-      add_new_record
-
-      list = subject.refresh_databases_list(connection)
-      expect(list.size).to be 4
-      expect(list).to include(:type => "standby", :host => "4.4.4.4", :dbname => "some_db",
-                              :user => "root", :active => true)
-    end
-  end
-
-  describe "#standby_databases" do
-    it "returns list of databases in standby mode" do
-      list = subject.standby_databases(connection)
-      expect(list.size).to eq 2
     end
   end
 
-  describe "#active_standby_databases" do
-    it "returns list of active databases in standby mode" do
-      list = subject.active_standby_databases(connection)
-      expect(list.size).to eq 1
-    end
+  def all_db_list
+    [{:type => 'master', :active => true, :host => '1.1.1.1', :user => 'root', :dbname => 'vmdb_test'},
```

Comment:
Try to use IP addresses specifically in the range designated for documentation (https://en.wikipedia.org/wiki/Reserved_IP_addresses)